sql/mysqld.cc

```diff
@@ -6173,7 +6174,10 @@ void create_new_thread(CONNECT *connect)
 
   uint sum= connection_count + extra_connection_count;
   if (sum > max_used_connections)
+  {
     max_used_connections= sum;
+    max_used_connections_time= time(nullptr);
```

Comment:
I haven't researched about the option itself, but just let me know the logic why reset here is important and how is different from `mysql_init_variables`?

Reply:
This isn't a reset - see `man 2 time`. Look at `threadpool_add_connection` - `thd->start_utime` already contains the time, so using this condition to set `max_used_connections*` that after `connect->scheduler->add_connection(connect);` which set this this would avoid a system call.

Reply:
> This isn't a reset - see `man 2 time`. Look at `threadpool_add_connection` - `thd->start_utime` already contains the time, so using this condition to set `max_used_connections*` that after `connect->scheduler->add_connection(connect);` which set this this would avoid a system call.

Thanks @grooverdan for the review, I addressed almost everything reviewers asked for except this particular comment - despite comments for the function `create_new_thread()` it does not contains `thd` parameter - the signature is `void create_new_thread(CONNECT *connect)` Socan you give me a hint how can I get `start_utime` parameter here?

Reply:
`connect->prior_thr_create_utime` doesn't seem to work, so leave as is with `time`